src/main/java/de/worldiety/autocd/util/FileType.java

```diff
@@ -8,6 +8,7 @@
     VUE("vue", "vue-builder", "vue-prod", "RUN npm run build\n"),
     NUXT("vue", "nuxt-builder", "nuxt-prod", "RUN npm i && npm run build"),
     EISEN("eisen", "vue-builder", "vue-prod", "RUN npm run build\n"),
+    SWIFT("swift", "swift-builder", "swift-prod", ""),
```

Comment:
Please supply a `defaultBuild` command here as the last argument.

AutoCD has a feature which allows to override the build process specified in the Dockerfile via a `build.sh` file in the project root. As seen [here](https://github.com/worldiety/AutoCD/blob/master/src/main/java/de/worldiety/autocd/docker/DockerfileHandler.java#L151-L157), we use the `defaultBuild` field if we don't find a 'build.sh'.  I recon that you could move the last line of your `swift-builder` Dockerfile into this field.

Reply:
Hi, I fixed it! Thanks :)